test/Query.js

```diff
@@ -674,56 +678,50 @@ describe('Query', function (){
   	});
   });
 
-  it('Should allow multiple local indexes and query correctly', function (done) {
+  it('Should allow multiple local indexes and query correctly', async function () {
     var schema = new dynamoose.Schema({
       id: {
         type: String,
         hashKey: true,
-        required: true
       },
       orgId: {
         type: String,
-        index: [{
-          global    : false,
-          name      : 'OrganizationCreateAtIndex',
-          rangeKey  : 'createdAt',
-          throughput: 1
-      }, {
-          global    : false,
-          name      : 'OrganizationExpectedArriveAtIndex',
-          rangeKey  : 'expectedArriveAt',
-          throughput: 1
-      }],
-      required: true,
+        rangeKey: true,
+      },
+      updatedAt: {
+        type: Date,
+        index: {
+          global: false,
+          name  : 'OrganizationUpdatedAtIndex'
+        }
       },
-      expectedArriveAt: Date
+      expectedArriveAt: {
+        type: Date,
+        index: {
+          global: false,
+          name  : 'OrganizationExpectedArriveAtIndex'
+        }
+      }
     },{
       throughput: 1,
       timestamps: true
     });
-    var Log = dynamoose.model('Log-1', schema);
+    var Log = dynamoose.model('Log-2', schema);
 
     var log1 = new Log({id: "test1", orgId: "org1", expectedArriveAt: Date.now()});
-    log1.save(function() {
-      Log.query('orgId').eq("org1")
+    var log2 = new Log({id: "test1", orgId: "org2", expectedArriveAt: Date.now()});
+
+    await log1.save();
+    await log2.save();
+
+    var res = await Log.query('id').eq("test1")
       .where('expectedArriveAt').lt( new Date() )
       .exec()
-      .then(function(res){
-        res.length.should.eql(1);
-        Log.query('orgId').eq("org1")
-        .where('createdAt').lt( new Date() )
-        .exec()
-        .then(function(res){
-          res.length.should.eql(1);
-          done();
-        })
-        .catch(function(e){
-          done(e);
-        });
-      })
-      .catch(function(e){
-        done(e);
-      });
-    });
+    res.length.should.eql(2);
+    
+    var res2 = await Log.query('id').eq("test1")
```

Comment:
And finally, same thing here related to previous comments.

Reply:
I took a look at #343 before making my changes. It's about querying multiple global indexes. There were two tests written for that issue though: one with global indexes, one with local. I didn't make changes to the one with global indexes, so #343 is still getting tested. As I explained above, the test that I updated never actually tested what it was supposed to test, instead it was likely acting as a duplicate, because it failed to overwrite the indexes on the table created by that first test. To see what I mean, try changing the name of the model from Log-1 to something else and re-running the test suite. This test will fail.

Reply:
@dolsem Got it